### PR TITLE
Add support for OpenAI TTS Engine

### PR DESCRIPTION
**Pull request by Ashesh3**
Required for #431
Frontend: https://github.com/coronasafe/ayushma_fe/pull/177

This pull request adds support for the OpenAI TTS Engine by introducing a new migration file, updating the project model, and modifying the language_helpers and openaiapi files to handle the new engine.

### CODE REVIEW

File: ayushma/models/project.py [hunk 1]
```diff
@@ -16,6 +16,9 @@ class Project(BaseModel):
     stt_engine = models.IntegerField(
         choices=STTEngine.choices, default=STTEngine.WHISPER
     )
+    tts_engine = models.IntegerField(
```
Comment (rithviknishad) [inline ayushma/models/project.py:19]: SmallInteger would be more suitable here

File: ayushma/models/enums.py [hunk 2]
```diff
@@ -19,6 +19,11 @@ class STTEngine(IntegerChoices):
     SELF_HOSTED = 3
 
 
+class TTSEngine(IntegerChoices):
+    OPENAI = 1
+    GOOGLE = 2
```
Comment (rithviknishad) [inline ayushma/models/enums.py:24]:
```suggestion
class TTSEngine(IntegerChoices):
    OPENAI = 1
    GOOGLE = 2
```
```suggestion
class TTSEngine(IntegerChoices):
    OPENAI = (1, "openai")
    GOOGLE = (2, "google")
```

File: ayushma/utils/language_helpers.py [hunk 3]
```diff
@@ -37,31 +39,43 @@ def sanitize_text(text):
...
-
-        response = client.synthesize_speech(
-            input=synthesis_input,
-            voice=voice,
-            audio_config=audio_config,
-        )
-
-        return response.audio_content
+
+        if service == "google":
```
Comment (rithviknishad) [inline ayushma/utils/language_helpers.py:50]:
```suggestion
        if service == TTSEngine.GOOGLE:
```

File: ayushma/utils/language_helpers.py [hunk 4]
```diff
@@ -37,31 +39,43 @@ def sanitize_text(text):
...
+            )
+
+            response = client.synthesize_speech(
+                input=synthesis_input,
+                voice=voice,
+                audio_config=audio_config,
+            )
+
+            return response.audio_content
+        elif service == "openai":
```
Comment (rithviknishad) [inline ayushma/utils/language_helpers.py:69]:
```suggestion
        elif service == TTSEngine.OPENAI:
```

File: ayushma/models/project.py (hunk 1 above)
Comment (Ashesh3) [reply ayushma/models/project.py]: Agreed

**Review by rithviknishad — APPROVED** [review]

File: ayushma/utils/language_helpers.py [hunk 5]
```diff
@@ -37,31 +41,43 @@ def sanitize_text(text):
     return sanitized_text
 
 
-def text_to_speech(text, language_code):
+def text_to_speech(text, language_code, service):
     try:
         # in en-IN neural voice is not available
         if language_code == "en-IN":
             language_code = "en-US"
```
Comment (sainak) [inline ayushma/utils/language_helpers.py:49]: is this applicable to both engines?

Comment (Ashesh3) [reply ayushma/utils/language_helpers.py:49]: `language_code` is picked up only by Google's engine and has no effect on openai. So, it should be fine.

Comment (sainak) [reply ayushma/utils/language_helpers.py:49]: okay

**Review by sainak — APPROVED** [review]

**Pull request merged**